RoT/Protocol/Challenge_Protocol.md

```diff
@@ -1,5688 +1,2854 @@
-   </td>
-   <td>8 bits
-   </td>
-  </tr>
-  <tr>
-   <td>Source Endpoint Id
-   </td>
-   <td>The EID of the originator of the MCTP packet
-   </td>
-   <td>8 bits
-   </td>
-  </tr>
-  <tr>
-   <td>SOM
-   </td>
-   <td>Start of Message is set to true (1b) for the first packet of a message.
-   </td>
-   <td>1 bit
-   </td>
-  </tr>
-  <tr>
-   <td>EOM
-   </td>
-   <td>End of Message is set to true (1b) for the last packet of a message.
-   </td>
-   <td>1 bit
-   </td>
-  </tr>
-  <tr>
-   <td>Pkt Seq#
-   </td>
-   <td>Packet Sequence Number for messages that span multiple packets.  Increments modulo 4 on each successive packet up through the packet contained the EOM flag set. 
-   </td>
-   <td>2 bits
-   </td>
-  </tr>
-  <tr>
-   <td>Message Tag
-   </td>
-   <td>Combined with Source Endpoint Id and TO field to identify unique message at MCTP transport layer.   
-<p>
-For messages that are split up into multiple packets, the TO and Message Tag bits remain the same for all packets from the SOM to the EOM.
-   </td>
-   <td>3 bits
-   </td>
-  </tr>
-  <tr>
-   <td>TO
-   </td>
-   <td>Tag Owner bit identifies whether the message tag was originated by the endpoint that is the source of the message or by the endpoint that is the destination of the message.  MCTP message types use this for Request/Response messages.
-   </td>
-   <td>1 bit
-   </td>
-  </tr>
-  <tr>
-   <td>Message body
-   </td>
-   <td>Payload of the MCTP message, can span multiple MCTP packets
-   </td>
-   <td>Variable
-   </td>
-  </tr>
-  <tr>
-   <td>IC
-   </td>
-   <td>MCTP Integrity check bit
-<p>
-0 = No MCTP message integrity
-<p>
-1 = MCTP message integrity check is present
-   </td>
-   <td>1 bit
-   </td>
-  </tr>
-  <tr>
-   <td>Message Type
-   </td>
-   <td>Defines the type of payload within the MCTP message header and data.  Message type codes are defined in the MCTP ID and Codes
-   </td>
-   <td>7 bits
-   </td>
-  </tr>
-  <tr>
-   <td>Message header
-   </td>
-   <td>Header data for the message type.  
-   </td>
-   <td>Variable
-   </td>
-  </tr>
-  <tr>
-   <td>Message Data
-   </td>
-   <td>Data for the message defined by the message type
-   </td>
-   <td>Variable
-   </td>
-  </tr>
-  <tr>
-   <td>MCTP Packet Payload
-   </td>
-   <td>Payload of the message body carried in the packet.  Limited by the transfer unit size.  Review MCTP Base Specification for further details.
-   </td>
-   <td>Variable
-   </td>
-  </tr>
-  <tr>
-   <td>Message Integrity Check
-   </td>
-   <td>Message type specific integrity check over the contest of the message body
-   </td>
-   <td>Variable
-   </td>
-  </tr>
-</table>
+    Table 2 Vendor Defined Message
 
 
-Null (0) Source and Destination EIDs are typically supported, however AC-RoT devices that have multiple MCTP Endpoints may specify an EID value greater than 7 and less than 255.  The PA-RoT does not broadcast any MCTP messages.
+| -------------- | ---- | --- |
```

Comment:
This doesn't seem to render well in github's viewer.  Is that a viewer-specific issue or there something else going on here.

Reply:
This was a formatting mistake on my part. Good catch.